Configuration/TypoScript/Configuration/Domains.typoscript

```diff
@@ -0,0 +1,32 @@
+headless_domains = PAGE
+headless_domains {
+    typeNum = 1608564571
+
+    config {
+        no_cache = 0
+        sendCacheHeaders = 1
+        debug = 0
+        admPanel = 0
+        disableAllHeaderCode = 1
+        additionalHeaders.10 {
+            header = Content-Type: application/json; charset=utf-8
+            replace = 1
+        }
+    }
+
+    10 = JSON
+    10 {
+        fields {
+            domains = TEXT
+            domains {
+                dataProcessing {
```

Comment:
maybe try to run dataProcessing without domains = TEXT? dataProcessing is available for JSON content object without using additional content objects

Reply:
Simplified version in new commit